### PR TITLE
Mark as read does not work in notifications

### PR DESCRIPTION
**Pull request by martins87**
## Link to Issue
Closes: #3136 

## Description of Changes
- Edits style for new notifications (those unread)
- Mark as read working "real time"

image

image

## Test Plan
- CA (click around) tested on local
- Design QA to see if it looks nice
- To help see notifications:
1. Log in with an account, make comments on thread (or like it)
2. Developer tools > Empty cache and force reload
3. Log in with account that will receive notifications

### CODE REVIEW

File: packages/commonwealth/client/styles/pages/notifications/notification_row.scss [hunk 1]
```diff
@@ -81,11 +80,16 @@
...
 
   .comment-body-mark-as-read:hover {
     color: $primary-600;
   }
 }
+
+.NewNotificationRow {
+  background: $neutral-50;
+  border-bottom: 1px solid $neutral-300;
+}
```
Comment (masvelio) [inline packages/commonwealth/client/styles/pages/notifications/notification_row.scss:95]: we have class `unread` above in `NotificationRow`  - isn't it the thing that should be handled in this PR? Is it necessary to create another class? If it is, follow the pattern, so someting like `&.new`

**Review by masvelio — CHANGES_REQUESTED** [review]
The [ticket](https://github.com/hicommonwealth/commonwealth/issues/3136) is not fully implemented 

https://user-images.githubusercontent.com/14819225/229043493-ee570101-e0d1-41b4-81f1-08a1fae01437.mp4

File: packages/commonwealth/client/styles/pages/notifications/notification_row.scss (hunk 1 above)
Comment (martins87) [reply packages/commonwealth/client/styles/pages/notifications/notification_row.scss]: Addressed

**Review by masvelio — APPROVED** [review]

**Comment by CowMuon** [comment]
Merged, but as discussed with Jake, need to document what our notifications roadmap looks like. @timolegros @NakulManchanda

**Pull request merged**